jetty-core/jetty-http3/jetty-http3-common/src/main/java/org/eclipse/jetty/http3/HTTP3StreamConnection.java

```diff
@@ -45,14 +46,21 @@ public abstract class HTTP3StreamConnection extends AbstractConnection
     private boolean useInputDirectByteBuffers = true;
     private HTTP3Stream stream;
     private RetainableByteBuffer inputBuffer;
+    private final int minBufferSpace;
```

Comment:
Move this `final` close to the other `final` fields and rename to `minInputBufferSpace` for consistency.